NOM_visualizations/python/nom_data.ipynb

```diff
@@ -0,0 +1,3353 @@
+{
```

Comment:
<ol><li>Maybe clarify <code>NMDC metadata endpoint</code> to <code>NMDC metadata collection_name endpoint</code> or something similar, since there are lots of metadata endpoints?</li><li>Also, I don't see a file called <code>nom_functions.py</code> in this folder? Maybe I am missing it. Or is it possible you mean the <code>nmdc_api.py</code> file?</li><li>I'd also recommend putting the get_next_results with tic marks since its a function name: <code>get_next_results</code>.</li><li>When you say "we can filter <code>data_object_set</code> by type "FT ICR-MS Analysis Results". I would rephrase because we have a <code>type</code> slot in the <code>DataObjectSet</code> which is not what you are using to filter, so this might be confusing. I would write the full slot name: "we can filter <code>data_object_set</code> by <code>data_object_type</code> using keyword search of "FT ICR-MS Analysis Results".</li><li>Can you write a short blurb about why you are grabbing the <code>id</code>,   <code>md5_checksum</code> and <code>url</code> fields? </li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microbiomedata/nmdc_notebooks/pull/68/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>